src/control/lib/control/telemetry.go

```diff
@@ -155,7 +171,13 @@ func MetricsQuery(ctx context.Context, req *MetricsQueryReq) (*MetricsQueryResp,
 		return nil, errors.New("port must be specified")
 	}
 
-	req.url = getMetricsURL(req.Host, req.Port)
+	if req.AllowInsecure == false && req.CaCertPath == "" {
+		return nil, errors.New("Provide the CA certificate path")
+	}
```

Comment:
This isn't right. The dmg config shouldn't need the user to supply a root cert if it is from a trusted CA that the OS recognizes.